glow/event_log.go

```diff
@@ -0,0 +1,112 @@
+package glow
+
+// This file contains a simple event logger, designed to collect a limited
+// number of timestamped logs and dump them to file or terminal.
+
+import (
+	"container/list"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+)
+
+// Event.
+type event struct {
+	ts   time.Time
+	line string
+}
+
+// Options for the event log.
+type EventLogOptions struct {
```

Comment:
A more helpful comment might be "EventLogOptions provides configuration for the EventLog, mainly targeted around managing the in-memory footprint of the EventLog"

Reply:
Point taken, comments need to be more verbose and descriptive.